src/Evolution/Systems/Cce/ReceiveTags.hpp

```diff
@@ -0,0 +1,34 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Variables.hpp"
+#include "Time/TimeStepId.hpp"
+
+namespace Cce {
+namespace ReceiveTags {
+
+namespace detail {
+struct AllDifferent {
```

Comment:
@nilsdeppe will put in a quick PR (hopefully today) making inserting something into an inbox more flexible so we can get rid of this struct...

Reply:
that sounds lovely. I was frustrated by the constraints that led to this workaround.

Reply:
I've submitted #1989, which should make this a lot easier (hopefully just having the inbox tag inherit off of one of the inserters added there)